source/index.html.slim

```diff
@@ -23,42 +23,35 @@ title: Home
             h1.hero__slogan
               span.hero__hidden KiwiRuby
               br
-              | The ruby conference you
```

Comment:
Couldn't think of a better tag line but this wasn't quite right so leaving this out for the moment